testkit/core/src/main/scala/com/lightbend/lagom/internal/testkit/TestkitSslSetup.scala

```diff
@@ -32,15 +32,17 @@ private[lagom] object TestkitSslSetup {
 
   /**
    *
-   * @param serverKeyStoreFile keyStore to setup the server
-   * @param trustStoreFile     trustStore for the clients
+   * @param keyStoreMetadata   keyStore to setup the server
+   * @param trustStoreMetadata trustStore for the clients
    * @param clientSslContext   SSLContext to create SSL clients
+   * @param sslPort            The port to bind to.
    * @return
    */
   def enabled(
       keyStoreMetadata: KeyStoreMetadata,
       trustStoreMetadata: KeyStoreMetadata,
-      clientSslContext: SSLContext
+      clientSslContext: SSLContext,
+      sslPort: Option[Int]
```

Comment:
```suggestion
      sslPort: Option[Int] = Some(0)
```

Adding the default value makes the change backwards compatible. You can also duplicate the methods and add a deprecation notice on the overload without `sslPort: Option[Int]`.

Also, if this method requires an sslPort, could the type of sslPort be `Int` and then add  a `require(sslPort > 1024 && sslPort < 65536)` on the implementation.

Reply:
hm, those requirements seem like they would also apply to the non-ssl port, although I was thinking that if someone is specifying a particular port we perhaps shouldn't care what port they chose. it does seem good to check if the `port != sslPort` though.

Reply:
> hm, those requirements seem like they would also apply to the non-ssl port

I'm considering the non-ssl port to be compulsory: "test MUST open a cleartext port and MAY open a TLS port".

Reply:
@ignasi35 i mean `require(sslPort > 1024 && sslPort < 65536)` -- doesn't that also apply to the non-SSL port?

Reply:
Oh, absolutely it does! I missed your point there.